Misc/NEWS.d/next/Tools/Demos/2019-10-02-09-48-42.bpo-38347.2Tq5D1.rst

```diff
@@ -0,0 +1 @@
+pathfix.py: Assume all files that end on '.py' are Python scripts when working recursively.
```

Comment:
I'm not sure how you created this file, but blurb uses the directory "Misc/NEWS.d/next/Tools-Demos/", not "Misc/NEWS.d/next/Tools/Demos/".

Reply:
Weird. It was created by blurb in d8e988efe85bfef307258d06bbeae72e4a21d1fe. Fixed in a17410299b88bb3063daaed18f67efe70c9f2a2c.